.vib/fluentd/goss/goss.yaml

```diff
@@ -0,0 +1,28 @@
+
+file:
+  /opt/bitnami/fluentd/Gemfile:
+    mode: "0644"
+    filetype: file
+    exists: true  
+command:
+  fluentd-conf-test:
+    exec: fluentd -c /opt/bitnami/fluentd/conf/fluentd.conf --dry-run
+    exit-status: 0
+    stdout: 
+      - {{ printf "/bitnami.*%s/" .Env.BITNAMI_APP_NAME }}
```

Comment:
Do we know what the command outputs when it fails? The checked stdout may be too general. I would personally also look for the `parsing ... succeded` words or something similar.

Reply:
Okay, added some more checks here.